eth/test/integration.js

```diff
@@ -152,12 +169,123 @@ contract("TreasureHuntCreator test", async accounts => {
     });
   });
 
+  describe("getLeaderboard", async () => {
+    it("should return an empty list if no players", async () => {
+      let testSolution1 = "A solution 1";
+      let solutionKey1 = await getSolutionAddress(testSolution1);
+
+      let testSolution2 = "A solution 2";
+      let solutionKey2 = await getSolutionAddress(testSolution2);
+
+      let testSolution3 = "A solution 3";
+      let solutionKey3 = await getSolutionAddress(testSolution3);
+
```

Comment:
do we need all these solutions and quests? for the sake of this test one (or maybe even 0?) should be enough. Just to keep the test clearer